src/app/detail/ui/Sidebar.tsx

```diff
@@ -111,17 +134,19 @@ export const Sidebar = ({
             disabled={disabled}
           />
         </Stack>
-        <Stack direction="row" m="16px 0px" alignItems="center">
+        <Stack direction="row" m="16px 0px" alignItems="center" justifyContent="space-between">
           <StyledText variant="md">Due Date</StyledText>
-          <DatePickerComponent
-            getDate={(date) => {
-              const isoDate = formatDate(date)
-              updateTask({
-                dueDate: isoDate,
-              })
-            }}
-            dateValue={dueDate ? isoToReadableDate(dueDate) : undefined}
-          />
+          <Box>
+            <DatePickerComponent
+              getDate={(date) => {
+                const isoDate = formatDate(date)
+                updateTask({
+                  dueDate: isoDate,
+                })
+              }}
+              dateValue={dueDate ? isoToReadableDate(dueDate) : undefined}
```

Comment:
```suggestion
              }}
              dateValue={!dueDate ?? isoToReadableDate(dueDate) }
```

This is optional as we would also have to handle condition for `null`

Reply:
The above suggestion doesn't work with the type of dateValue for the `DatePickerComponent`. 
`null` cases has already been handled.